pkg/cmd/admin/node/evacuate.go

```diff
@@ -98,26 +98,52 @@ func (e *EvacuateOptions) RunEvacuate(node *kapi.Node) error {
 		return err
 	}
 
+	rss, err := e.Options.Kclient.ReplicaSets(kapi.NamespaceAll).List(kapi.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	dss, err := e.Options.Kclient.DaemonSets(kapi.NamespaceAll).List(kapi.ListOptions{})
```

Comment:
pet sets added. @smarterclayton do you know if we have any helper/utility that knows how to match the pods using `MatchExpressions`?